### DISCUSSION TITLE
Upgrade from 2.0.2 to 2.0.3

### DISCUSSION THREAD

**Discussion by swapkh91** [linsoss/doris-operator#36] [General]
hi @Al-assad 

i want to upgrade my doris cluster running on 2.0.2 to 2.0.3
what would be the suggested steps to take so that current deployment doesn't break?
I'm planning on using these official images

https://hub.docker.com/r/selectdb/doris.be-ubuntu/tags
https://hub.docker.com/r/selectdb/doris.fe-ubuntu/tags

**Comment by Al-assad** [comment]
Unfortunately, https://hub.docker.com/r/selectdb/doris.be-ubuntu and  doris-operator are temporarily incompatible. However, I have recently released the image of doris 2.0.3， see：
- [doris-fe:2.0.3](https://github.com/linsoss/doris-operator/pkgs/container/doris-fe)
- [doris-be:2.0.3](https://github.com/linsoss/doris-operator/pkgs/container/doris-be)

**Reply by swapkh91** [reply]
got it, thanks 👍 
so to upgrade, do I just change the image version in my yaml and apply or is there some other way to do upgrade and without breaking current deployment?

**Reply by Al-assad** [reply]
YES, it's just that simple

**Reply by swapkh91** [reply]
Cool
